src/main/java/org/ideoholic/curium/model/account/service/AccountService.java

```diff
@@ -1152,16 +1152,25 @@ public boolean getIncomeStatement(IncomeStatementDto incomeStatementDto) {
 	}
 		return true;
 	}
+
+	//This method is placed here for MessSuppliersAction. Please delete this method after migrating MessSuppliersAction.
+	public boolean printSearchJournalEntries(){
+		PrintSearchJournalEntriesDto printSearchJournalEntriesDto = new PrintSearchJournalEntriesDto();
+		printSearchJournalEntriesDto.setAccountDetails(request.getParameter("accountidselected"));
+		printSearchJournalEntriesDto.setFromDate(request.getParameter("fromdateselected"));
+		printSearchJournalEntriesDto.setToDate(request.getParameter("todateselected"));
+
+		return printSearchJournalEntries(printSearchJournalEntriesDto);
```

Comment:
This method cannot be returned directly.
The below original setAttribute needs to be moved from the service class to the adapter.
- request.setAttribute("ledgertransactions", voucherMap);
- request.setAttribute("ledgername", accountIdName[1]);
- request.setAttribute("fromdateselected", request.getParameter("fromdateselected"));
- request.setAttribute("todateselected", request.getParameter("todateselected"));